content/post/dc-roma-ii-nix/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Nix on the DC ROMA II
+description: Getting Nix bootstrapped on a RISC-V laptop using Ubuntu.
+date: 2025-11-17 11:50:00 +0800
+categories:
+    - riscv
+---
+
+Back in September, I was at NixCon 2025. I was able to meet Yuning Liang, the founder of DeepComputing.
+We had previously connected on LinkedIn several months prior. I had ordered their new laptop, the DC ROMA II.
+It has an ESWIN EIC7700X, an SoC with 8 out of order execution RV64GBC cores. This is a huge upgrade compared
+to the 4 in-order execution cores on the JH7110 that the VisionFive 2 uses. After coming back from a trip
+to Portland on the 8th, I started looking into getting Nix & NixOS working on it. Surely since I am able
```

Comment:
Omit "After coming back from a trip to Portland on the 8th": Irrelevant information